scorm-tool/src/java/org/sakaiproject/scorm/ui/console/pages/PackageRemovePage.java

```diff
@@ -108,7 +108,7 @@ public FileRemoveForm( String id, final PageParameters params )
 			DataTable removeTable = new DataTable( "removeTable", columns, new ListDataProvider( list ), 3 );
 
 			final Label alertLabel = new Label( "alert", new ResourceModel( "verify.remove" ) );
-			final CancelButton btnCancel = new CancelButton( "btnCancel", PackageListPage.class );
+			final SakaiAjaxCancelButton btnCancel = new SakaiAjaxCancelButton( "btnCancel", PackageListPage.class );
```

Comment:
There is an IndicatingAjaxButton btnSubmit that should also be a spinner button right? I also notice that these buttons disable each other, which is not something the normal SakaiAjaxButton can do (it just worries about itself). Perhaps we should enhance it to allow it to disable other page elements so we don't have to keep adding the extra AjaxCallListener?

Reply:
Yes, good call on both. I've updated `SakaiAjaxButton` and its related `SakaiSpinnerAjaxCallListener` to take a list of element IDs which can be enabled/disabled on click.